rasa/core/training/story_conflict.py

```diff
@@ -242,6 +255,47 @@ def _find_conflicting_states(
     }
 
 
+def _unpredictable_action(event: Event) -> bool:
+    """Identifies if the action cannot be predicted by policies that use story data.
+
+    Args:
+        event: An event to be checked.
+
+    Returns:
+        A boolean indicating if the event can be predicted or not.
+    """
+    return event.__str__() == ACTION_UNLIKELY_INTENT_NAME
+
+
+def _find_unpredictable_actions(
+    trackers: List[TrackerWithCachedStates],
+    domain: Domain,
+    max_history: Optional[int],
+    tokenizer: Optional[Tokenizer],
+) -> Dict[int, Optional[List[Text]]]:
+    """Identifies all states that contain actions that cannot be predicted.
+
+    Args:
+        trackers: Trackers that contain the states.
+        domain: The domain object.
+        max_history: Number of turns to take into account for the state descriptions.
+        tokenizer: A tokenizer to tokenize the user messages.
+
+    Returns:
+        A dictionary mapping state-hashes to a list of actions that cannot be predicted.
```

Comment:
This might be nitpicking: doesn't this return a mapping of state-hases to a list of action-hashes, rather than actions?

Reply:
That's a good catch. Will update that!